research-software-directory/RSD-as-a-service
# 631: Upgrade next to v13

# Upgrade to Next v13

Closes #631 

Changes proposed in this pull request:
*  Upgrade frontend to Next v13     

How to test:
* `make start` to rebuild everything
* login to rsd and all functionality should work as previously :-)

PR Checklist:
